githubteacher-Oakland.md

```diff
@@ -0,0 +1,6 @@
+## Oakland :heart: :trophy: :trophy::trophy::trophy::trophy:
+
+#### Restaurants
+
+
+#### Things to do
```

Comment:
yeah brotha, thanks for the comment love! :)  Yes, you should come hang with me and grab a coffee at Kilovolt coffee in WEST OAKLAND.